Assets/Scripts/StuntSystem/Stunt.cs

```diff
@@ -0,0 +1,140 @@
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+
+public struct FrameKeyData
+{
+    public int frameCount;
+    public List<string> inputs;
+}
+
+public enum StuntType
+{
+    None,
+    Jump,
+    Spin,
+    Flip,
+    Grind
+}
+
+public class StuntDatabase
+{
+    public List<Stunt> stunts = new List<Stunt>();
+    public int frameDelayMax = 15;
+
+    public void PopulateStunts()
+    {
+        // Example of populating the stunt database
+        // If for efficiency, we can populate into a tree structure
+
+        // TODO: Add frame limit & priorities
+        // TODO: Single slides are broken atm. Absolutely needs priority
+
+        stunts.Add(new Stunt(4, "Frontside Pop Shuvit", StuntType.Flip, 2, 80, new string[] { "2", "3", "6", "StuntA" }));
+        stunts.Add(new Stunt(5, "Backside Pop Shuvit", StuntType.Flip, 2, 80, new string[] { "2", "1", "4", "StuntA" }));
+        stunts.Add(new Stunt(2, "Kickflip", StuntType.Flip, 1, 30, new string[] { "6", "StuntA" }));
+        stunts.Add(new Stunt(3, "Heelflip", StuntType.Flip, 1, 30, new string[] { "4", "StuntA" }));
+        stunts.Add(new Stunt(7, "Nose Slide", StuntType.Grind, 1, 50, new string[] { "4", "StuntB" }));
+        stunts.Add(new Stunt(8, "Tail Slide", StuntType.Grind, 1, 50, new string[] { "6", "StuntB" }));
+        stunts.Add(new Stunt(6, "50-50 Grind", StuntType.Grind, 1, 30, new string[] { "5", "StuntB" }));
+        // Add more stunts as needed
+    }
+
+    public Stunt QueryStuntByFrames(StuntType stuntType, List<FrameKeyData> data)
```

Comment:
This could be implemented as a tree structure (i.e. every "node" is a single input that branches downwards to the next input in the chain. This way I can search by "inputs" as opposed to iterating over and checking everything